src/types.ts

```diff
@@ -38,7 +38,7 @@ export interface ContextType {
   handleSearch: (input: string) => void
   layout: string
   libraryStatus: GameStatus[]
-  platform: string
+  platform: NodeJS.Platform | string
```

Comment:
Can you safely drop the `| string` or change it to `| 'unknown'`?

Reply:
No, process.platform always returns a | string so we should conserve it.

Reply:
@adityaruplaha yes
@dragonDScript no, because this here is the function we receive from the Main process, so we are not calling the process itself from the frontend since it is impossible.

Reply:
Remember, this is just a type, this should help us to understand what the function returns

Reply:
@adityaruplaha this doesnt work, it says can't accept a string, because 'unknown' is a string, I think its something with TS that I dont want to go deeper into.